packages/router/utils.ts

```diff
@@ -1533,11 +1533,17 @@ export const redirectDocument: RedirectFunction = (url, init) => {
   return response;
 };
 
+export interface ErrorResponse {
+  status: number;
+  statusText: string;
+  data: any;
+}
+
 /**
  * @private
  * Utility class we use to hold auto-unwrapped 4xx/5xx Response bodies
  */
```

Comment:
```suggestion
/**
 * @private
 * Utility class we use to hold auto-unwrapped 4xx/5xx Response bodies
 *
 * We don't export the class for public use since it's an implementation
 * detail, but we export the interface above so folks can build their own 
 * abstractions around instances via isRouteErrorResponse() 
 */
```